hyperspy/tests/io/test_protochips.py

```diff
@@ -75,47 +78,47 @@ def get_datetime(dt):
-    with nt.assert_raises(IOError) as cm:
+    with pytest.raises(IOError) as cm:
         ProtochipsCSV(filename)
-    nt.assert_equal(cm.exception.args[0], invalid_file_error)
+        cm.match(invalid_file_error)
 
 
 def test_loading_invalid_protochips_file():
     filename = os.path.join(dirpath, 'invalid_protochips_file.csv')
-    with nt.assert_raises(IOError) as cm:
+    with pytest.raises(IOError) as cm:
         hs.load(filename)
-    nt.assert_equal(cm.exception.args[0], invalid_file_error)
+        cm.match(invalid_file_error)
 
 
 class test_ProtochipsGasCellCSV():
 
-    def setUp(self):
+    def setup_method(self, method):
         filename = os.path.join(dirpath, 'protochips_gas_cell.csv')
         self.s_list = hs.load(filename)
 
     def test_read_metadata(self):
         date, time, dt_np = datetime_gas_cell
         for s in self.s_list:
-            nt.assert_equal(s.metadata.General.date, date)
-            nt.assert_equal(s.metadata.General.time, time)
-            nt.assert_equal(s.axes_manager[0].units, 's')
-            nt.assert_almost_equal(s.axes_manager[0].scale, 0.25995, places=5)
-            nt.assert_equal(s.axes_manager[0].offset, 0)
+            assert s.metadata.General.date == date
+            assert s.metadata.General.time == time
+            assert s.axes_manager[0].units == 's'
+            assert_allclose(s.axes_manager[0].scale, 0.25995, atol=1E-5)
+            assert s.axes_manager[0].offset == 0
 
     def test_read_original_metadata(self):
         om = self.s_list[0].original_metadata.Protochips_header
-        nt.assert_equal(om.Calibration_file_path, "The calibration files names"
-                        " are saved in the 'Original notes' array of the "
-                        "original metadata.")
-        nt.assert_equal(om.Holder_Pressure_units, 'Torr')
-        nt.assert_equal(om.Holder_Temperature_units, 'Degrees C')
-        nt.assert_equal(om.Start_time, datetime_gas_cell[2])
-        nt.assert_equal(om.Holder_Pressure_units, 'Torr')
-        nt.assert_equal(om.Tank1_Pressure_units, 'Torr')
-        nt.assert_equal(om.Tank2_Pressure_units, 'Torr')
-        nt.assert_equal(om.Vacuum_Tank_Pressure_units, 'Torr')
-        nt.assert_equal(om.Time_units, 'Milliseconds')
-        nt.assert_equal(om.User, 'eric')
+        assert (om.Calibration_file_name == "The calibration files names"
+                " are saved in the 'Original notes' array of the "
+                "original metadata.")
+        assert om.Holder_Pressure_units == 'Torr'
+        assert om.Holder_Temperature_units == 'Degrees C'
+        assert om.Start_time == datetime_gas_cell[2]
+        assert om.Holder_Pressure_units == 'Torr'
+        assert om.Tank1_Pressure_units == 'Torr'
+        assert om.Tank2_Pressure_units == 'Torr'
+        assert om.Vacuum_Tank_Pressure_units == 'Torr'
+        assert om.Time_units == 'Milliseconds'
+        assert om.User == 'eric'
 
 
 class test_ProtochipsGasCellCSVNoUser():
```

Comment:
This part haven't been updated and these tests are not collected (and run) by pytest, possibly because they need to be renamed to `TestProtochipsGasCellCSVNoUser`. There are a few other one, below.